[2023-03-28 13:49:56,298] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: postgres_etl_dag.etl_from_radb 2023-03-26T00:00:00+00:00 [queued]>
[2023-03-28 13:49:56,361] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: postgres_etl_dag.etl_from_radb 2023-03-26T00:00:00+00:00 [queued]>
[2023-03-28 13:49:56,364] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2023-03-28 13:49:56,365] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2023-03-28 13:49:56,367] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2023-03-28 13:49:56,501] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): etl_from_radb> on 2023-03-26T00:00:00+00:00
[2023-03-28 13:49:56,575] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_etl_dag', 'etl_from_radb', '2023-03-26T00:00:00+00:00', '--job-id', '15', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmpatotpnq_', '--error-file', '/tmp/tmp1llciq3w']
[2023-03-28 13:49:56,582] {standard_task_runner.py:77} INFO - Job 15: Subtask etl_from_radb
[2023-03-28 13:49:56,559] {standard_task_runner.py:52} INFO - Started process 2855 to run task
[2023-03-28 13:49:56,845] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_etl_dag.etl_from_radb 2023-03-26T00:00:00+00:00 [running]> on host 0bc3da97cebb
[2023-03-28 13:49:57,564] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_etl_dag
AIRFLOW_CTX_TASK_ID=etl_from_radb
AIRFLOW_CTX_EXECUTION_DATE=2023-03-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-26T00:00:00+00:00
[2023-03-28 13:50:00,929] {python.py:151} INFO - Done. Returned value was: [(111, 111, 'test'), (111, 111, 'test'), (111, 111, 'test'), (111, 111, 'test'), (111, 111, 'test'), (111, 111, 'test'), (111, 111, 'test'), (111, 111, 'test'), (111, 111, 'test'), (111, 111, 'test'), (111, 111, 'test')]
[2023-03-28 13:50:05,436] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=postgres_etl_dag, task_id=etl_from_radb, execution_date=20230326T000000, start_date=20230328T134956, end_date=20230328T135005
[2023-03-28 13:50:06,014] {local_task_job.py:151} INFO - Task exited with return code 0
[2023-03-28 13:50:06,218] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
